configure.sh

```diff
@@ -53,3 +53,19 @@ write_action_env_to_bazelrc "TF_HEADER_DIR" ${TF_CFLAGS:2}
 write_action_env_to_bazelrc "TF_SHARED_LIBRARY_DIR" ${SHARED_LIBRARY_DIR}
 write_action_env_to_bazelrc "TF_SHARED_LIBRARY_NAME" ${SHARED_LIBRARY_NAME}
 write_action_env_to_bazelrc "TF_CXX11_ABI_FLAG" ${TF_CXX11_ABI_FLAG}
+
+write_to_bazelrc "build:cuda --define=using_cuda=true --define=using_cuda_nvcc=true"
+write_to_bazelrc "build:cuda --crosstool_top=@local_config_cuda//crosstool:toolchain"
+write_to_bazelrc "build --spawn_strategy=standalone"
+write_to_bazelrc "build --strategy=Genrule=standalone"
+write_to_bazelrc "build -c opt"
+write_action_env_to_bazelrc "TF_NEED_CUDA" ${TF_NEED_CUDA}
+# TODO(yifeif): do not hardcode path
+if [[ "$TF_NEED_CUDA" == "1" ]]; then
+  write_action_env_to_bazelrc "CUDNN_INSTALL_PATH" "/usr/lib/x86_64-linux-gnu"
+  write_action_env_to_bazelrc "TF_CUDA_VERSION" "10.0"
+  write_action_env_to_bazelrc "TF_CUDNN_VERSION" "7"
+  write_action_env_to_bazelrc "CUDA_TOOLKIT_PATH" "/usr/local/cuda"
```

Comment:
How do we feel about using $CUDA_HOME here?

Reply:
Hmmm seems custom-op-gpu doesn't have this ENV variable though I do think it's pretty standard practice:
https://numba.pydata.org/numba-doc/dev/cuda/overview.html#setting-cuda-installation-path

Reply:
Agree, I'll fix them later after we fix the environment issues :-)

Reply:
I leave a TODO comment here, and want to solve the problem in the following PRs. What do you think?